src/patterns.md

```diff
@@ -756,6 +761,57 @@ Path patterns are irrefutable when they refer to structs or an enum variant when
 has only one variant or a constant whose type is irrefutable. They are refutable when they
 refer to refutable constants or enum variants for enums with multiple variants.
 
+## Or-patterns
+
+_Or-patterns_ are patterns that match on either of two sub-patterns (e.g. `A |
+B`). They can nest arbitrarily. Syntactically, or-patterns are allowed in any
```

Comment:
I'm not sure I follow what this explicitly mentions "two" sub-patterns.  Can't it be any number?

Reply:
I had copy-pasted from the RFC without reading carefully. Fixed.